openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -29,11 +30,12 @@ def get_current_session_fps():
     session_fps = float(legacy_io.Session.get('AVALON_FPS', 25))
     return convert_to_maya_fps(session_fps)
 
+
 class ArnoldStandinLoader(load.LoaderPlugin):
     """Load as Arnold standin"""
 
-    families = ["ass", "animation", "model", "proxyAbc", "pointcache"]
-    representations = ["ass", "abc"]
+    families = ["ass", "animation", "model", "proxyAbc", "pointcache", "usd"]
+    representations = ["ass", "abc", "usda", "usdc", "usd"]
```

Comment:
I wonder if all of these extensions `usda` and `usdc` should actually be representation `usd`?
```suggestion
    representations = ["ass", "abc", "usd"]
```

Reply:
What is `usda` and `usdc`?

Reply:
> USDA files are plain ascii, while USDC are binary files

Also see [here](https://openusd.org/release/tut_converting_between_layer_formats.html#converting-between-usda-usdc-and-usd)